TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.CfgAfterIterativePerformProcedureTransformer.cs

```diff
@@ -47,6 +56,12 @@ private bool Callback(BasicBlock<Node, D> block, int incomingEdge, BasicBlock<No
                     int entranceEdge = cfg.SuccessorEdges.Count;
                     cfg.SuccessorEdges.Add(iterativeGroup.Group.First.Value);
                     block.SuccessorEdges.Add(entranceEdge);
+                    System.Diagnostics.Debug.Assert(block.Context == null);
+                    //If the assert above fails (it should not be case), then it will be necessay to exeute the commented code bellow.
```

Comment:
typos: If the assert above fails (it should not be _the_ case), then it will be _necessary_ to _execute_ the commented code _below_.

Reply:
done in 0067e69